crates/transaction-generator-lib/src/lib.rs

```diff
@@ -290,6 +314,24 @@ pub async fn create_txn_generator_creator(
                         *batch_size,
                     ))
                 },
+                TransactionType::Workflow {
+                    num_modules,
+                    use_account_pool,
+                    workflow_kind,
+                    move_stages_by_phase,
+                } => Box::new(
+                    WorkflowTxnGeneratorCreator::create_workload(
+                        *workflow_kind,
+                        txn_factory.clone(),
+                        init_txn_factory.clone(),
+                        root_account,
+                        txn_executor,
+                        *num_modules,
+                        use_account_pool.then(|| accounts_pool.clone()),
```

Comment:
Where are we specifying that the output of previous phase is to be used as input to this phase?